### PR TITLE
adding common and preparing page people

### PR DESCRIPTION
**Pull request by Mateusszzp**
Please, did I understand my task correctly added page with popular people without navigation connected to API

### CODE REVIEW

**Review by KubaTuta — CHANGES_REQUESTED** [review]
yep, you should mapped only 1 tile on api or at least at exapmple api from /public

**Comment by Mateusszzp** [comment]
> yep, you should mapped only 1 tile on api or at least at exapmple api from /public

hey, it seems to me that it is made so universally that it can be added as needed and downloaded from the api

File: src/store.js [hunk 1]
```diff
@@ -2,12 +2,13 @@ import createSagaMiddleware from '@redux-saga/core';
...
 import rootSaga from './rootSaga';
 import popularMoviesReducer from './Features/Movies/popularMoviesSlice';
-
+import popularPeopleReduser from './Features/People/popularPeopleSlice';
 const sagaMiddleware = createSagaMiddleware();
 
 const store = configureStore({
   reducer: {
     popularMovies: popularMoviesReducer,
+    popularPeople: popularPeopleReduser,
```
Comment (HaniaNassalska) [inline src/store.js]: I think it should be popularPeopleReducer :)

**Review by HaniaNassalska — APPROVED** [review]

File: src/store.js (hunk 1 above)
Comment (Mateusszzp) [reply src/store.js]: Yes you are right, I agree with you 100%.

**Pull request closed without merging**